src/templates/ja/2019/methodology.html

```diff
@@ -0,0 +1,432 @@
+      </table>
+      </div></div>
+
+      <p class="note">
+        * 注 Lighthouseは通常、CPUが4倍遅くなるように設定されていますが、WebPageTestの<a href="https://github.com/GoogleChrome/lighthouse/issues/9668#issuecomment-535134232">バグ</a>のため、テスト時には1倍になっていました。
+      </p>
+
+      <p>
+        LighthouseとHTTP Archiveで利用可能な監査の詳細については、<a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse開発者向けドキュメント</a>を参照してください。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="wappalyzer"><a href="#wappalyzer" class="anchor-link">Wappalyzer</a></h3>
+
+      <p>
+        <a href="https://www.wappalyzer.com/">Wappalyzer</a>は、ウェブページで使用されている技術を検出するためのツールです。<a href="https://www.wappalyzer.com/technologies">65のカテゴリ</a>があり、JavaScriptフレームワークからCMSプラットフォーム、さらには暗号通貨の採掘者に至るまで、テストされた技術の範囲があります。1,200以上の技術がサポートされています。
+      </p>
+
+      <p>
+        HTTP Archiveは、すべてのウェブページに対して最新バージョンのWappalyzerを実行します。2019年7月現在、Web AlmanacはWappalyzerの<a href="https://github.com/AliasIO/Wappalyzer/releases/tag/v5.8.3">5.8.3バージョン</a>を使用しています。
+      </p>
+
+      <p>
+        Wappalyzerは、WordPress、Bootstrap、jQueryのような開発者ツールの人気を分析する多くの章を強力にしています。例えば、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='ecommerce')}}">Eコマース</a> と <a href="{{ url_for('chapter', year=year, lang=lang, chapter='cms')}}">CMS</a> の各章は、それぞれの <a href="https://www.wappalyzer.com/categories/ecommerce">Eコマース</a> と <a href="https://www.wappalyzer.com/categories/cms">CMS</a> の各チャプターは、Wappalyzer が検出した技術のカテゴリに大きく依存しています。
+      </p>
+
+      <p>
+        Wappalyzerを含むすべての検出ツールには限界があります。その結果の妥当性は、その検出メカニズムがどれだけ正確であるかに常に依存します。Web Almanacでは、Wappalyzerが使用されているすべての章に注意書きが追加されますが、特定の理由により、その分析が正確でない場合があります。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="chrome-ux-report"><a href="#chrome-ux-report" class="anchor-link">Chrome UXレポート</a></h3>
+
+      <p>
+        <a href="https://developers.google.com/web/tools/chrome-user-experience-report">Chrome UXレポート</a> は、実際のChromeユーザーの体験をまとめた公開データセットです。エクスペリエンスは、<code>https://www.example.com</code> などのように、ウェブサイトの起源によってグループ化されています。このデータセットには、ペイント、ロード、インタラクション、レイアウトの安定性などのUXメトリクスの分布が含まれています。月ごとのグループ化に加えて、国レベルの地理、フォームファクター（デスクトップ、モバイル、タブレット）、有効な接続タイプ（4G、3Gなど）などの大きさ、速度によっても経験をスライスすることができます。
+      </p>
+
+      <p>
+        Chrome UXレポートの実世界のユーザー体験データを参照するWeb Almanacメトリクスについては、2019年7月のデータセット（201907）を使用しています。
+      </p>
+
+      <p>
+        データセットの詳細については、<a href="https://web.dev/chrome-ux-report-bigquery">BigQueryでChrome UXレポートを使用する</a>のガイド<a href="https://web.dev/">web.dev</a>を参照してください。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="third-party-web"><a href="#third-party-web">サードパーティウェブ</a></h3>
+
+      <p>
+        <a href="https://www.thirdpartyweb.today/">サードパーティウェブ</a>は、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='patrickhulce')}}">Patrick Hulce</a>の研究プロジェクトで、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='サードパーティ')}}">サードパーティ</a>の章では、HTTP ArchiveとLighthouseのデータを使用して、サードパーティのリソースがウェブに与える影響を特定して分析しています。
```

Comment:
Spotted one more error:
```suggestion
        <a href="https://www.thirdpartyweb.today/">サードパーティウェブ</a>は、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='patrickhulce')}}">Patrick Hulce</a>の研究プロジェクトで、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='third-parties')}}">サードパーティ</a>の章では、HTTP ArchiveとLighthouseのデータを使用して、サードパーティのリソースがウェブに与える影響を特定して分析しています。
```

Reply:
Sorry...

I fixed.